pokemongo_bot/cell_workers/recycle_items.py

```diff
@@ -77,9 +77,8 @@ def work(self):
         :rtype: WorkerResult
         """
         # TODO: Use new inventory everywhere and then remove the inventory update
-        # Updating inventory
-        inventory.refresh_inventory()
```

Comment:
This refresh_inventory call is to make sure that this worker will try to recycle items that actually are in the inventory. We need it until every task update the inventory when looting/using an item.